### PR TITLE
media-libs/opencv: add 4.9.0

### PR DESCRIPTION
**Pull request by negril**
media-libs/quirc: new package, add 1.2, 9999

Closes: https://bugs.gentoo.org/921968
Closes: https://bugs.gentoo.org/924005

### CODE REVIEW

File: profiles/arch/x86/use.stable.mask [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in use.mask
 
+# Paul Zander <negril.nx+gentoo@gmail.org> (2024-02-14)
+# sci-libs/atlas is not stable
+atlas
+# media-libs/jasper is not stable
```
Comment (thesamesam) [inline profiles/arch/x86/use.stable.mask:10]: Please move the comment line above as it's not well-formed right now.

File: media-libs/quirc/quirc-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+inherit multilib-minimal
+
+if [[ ${PV} = *9999* ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/dlbeer/${PN}.git"
+else
+	SRC_URI="
+		https://github.com/dlbeer/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
```
Comment (thesamesam) [inline media-libs/quirc/quirc-9999.ebuild:18]: Please do a keywording bug once it's merged.

File: media-libs/opencv/opencv-4.9.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,1017 @@
...
+	fi
+	if multilib_is_native_abi && use python; then
+		python_foreach_impl cmake_src_install
+		python_foreach_impl python_optimize
+	else
+		cmake_src_install
+	fi
+}
+
+multilib_src_test() {
```
Comment (thesamesam) [inline media-libs/opencv/opencv-4.9.0.ebuild:935]: Please don't do newline after function starts.

Also, multilib_src_test should go before multilib_src_install to reflect phase execution order.

**Review by thesamesam — CHANGES_REQUESTED** [review]
Left some small comments but they apply elsewhere too.

**Comment by Jamim** [comment]
_Hello @negril,_

Would you mind updating this PR in order to address review comments?
Please let me know if any help is needed.

_Thanks!_

**Pull request closed without merging**